pyccl/tracers.py

```diff
@@ -744,7 +752,39 @@ def CMBLensingTracer(cosmo, z_source, n_samples=100):
     return tracer
 
 
-def tSZTracer(cosmo, z_max=6., n_chi=1024):
+def ZPowerTracer(cosmo, *, A, alpha, z_min=0., z_max=6., n_chi=1024):
```

Comment:
I tested it and it won't complain if `z_max` < `z_min`. Can the C level handle this? If not, should we include a check?

Reply:
Checking now